WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -475,85 +475,80 @@ class AbstractPostListViewController: UIViewController,
         return .any
     }
 
-    func syncHelper(_ syncHelper: WPContentSyncHelper, syncContentWithUserInteraction userInteraction: Bool, success: ((_ hasMore: Bool) -> ())?, failure: ((_ error: NSError) -> ())?) {
+    @MainActor
+    func syncPosts(isFirstPage: Bool) async throws -> ([AbstractPost], Bool) {
```

Comment:
From the function signature, I couldn't understand what that `Bool` in the returned tuple represents. I only figured it out by looking at the usage, where it's assigned to `hasMore`.

Have you considered defining a `struct` to model that information?

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/22070/commits/5981dcff18b7e203f61deb654cfb57ac2f443e82. I used a named tuple instead of a struct.